xarray/tests/test_plot.py

```diff
@@ -2638,6 +2717,16 @@ def test_ncaxis_notinstalled_line_plot(self):
     DataArray(easy_array((10, 3, 2))),
 ]
 
+# Define a few arrays for testing scale="log"
+test_da_list_log = [
```

Comment:
@santisoler please could we make this into a fixture? If it's in the base of the module, then it'll be run on every import of this file, which will slow down collecting tests (even those that don't use it)

Reply:
I've just transformed the `test_da_list_log` as a fixture method in `TestAxesKwargs` with the number of dimensions of the returned DataArray as a parameter of the fixture. I took the liberty to do the same with the preexisting `test_da_list `.